<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkinUploadDialog</class>
 <widget class="QDialog" name="SkinUploadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Загрузка скина</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/skin.png</normaloff>:/resources/skin.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="formLayout">
     <item row="2" column="1">
      <widget class="QLineEdit" name="pathEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="nickLabel">
       <property name="text">
        <string>Игровое имя</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="passLabel">
       <property name="text">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="newpassLabel">
       <property name="text">
        <string>Скин</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="browseButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="passEdit"/>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="nickEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="messageFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="messageLabel">
        <property name="text">
         <string>Заполните форму и нажмите кнопку &quot;Загрузить&quot;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="downLayout">
     <item>
      <spacer name="hspc">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>Загрузить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>passEdit</tabstop>
  <tabstop>pathEdit</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>sendButton</tabstop>
  <tabstop>nickEdit</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
